contrib/update-genesis-validators/cmd/update_god_committee.go

```diff
@@ -0,0 +1,136 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authTypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	bankTypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	committeetypes "github.com/kava-labs/kava/x/committee/types"
+)
+
+const (
+	GOD_COMMITTEE_DESCRIPTION = "Kava God Committee (testing only)"
+)
+
+func InjectGodCommitteeMember(codec codec.Codec, appState genutiltypes.AppMap, memberAddress string) (genutiltypes.AppMap, error) {
+	// unmarshal auth module state
+	authState := authTypes.GenesisState{}
+	if err := codec.UnmarshalJSON(appState[authTypes.ModuleName], &authState); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal app_state.auth: %s", err)
+	}
+
+	// check to see if member's account exists already
+	var memberAccountExist bool
+	// assumes it is a simple base account
+	for _, rawAccount := range authState.Accounts {
+		accountValue := rawAccount.GetCachedValue()
+
+		var account *authTypes.BaseAccount
+
+		switch accountValue.(type) {
+		case *authTypes.BaseAccount:
+			account = accountValue.(*authTypes.BaseAccount)
+			if account.Address == memberAddress {
+				memberAccountExist = true
+				break
+			}
+		}
+	}
+
+	// if not create the members account
+	if !memberAccountExist {
+		fmt.Printf("member's %s account not found, creating it\n", memberAddress)
+
+		account := &authTypes.BaseAccount{
+			Address: memberAddress,
+		}
+
+		// convert back to any so we can store it in genesis state
+		memberAccount := authTypes.GenesisAccount(account)
+
+		anyAccounts, err := authTypes.PackAccounts([]authTypes.GenesisAccount{memberAccount})
+
+		if err != nil {
+			return nil, fmt.Errorf("failed to pack god committee: %s", err)
+		}
+
+		authState.Accounts = append(authState.Accounts, anyAccounts...)
+
+		// re-marshal updated state
+		appState[authTypes.ModuleName], err = codec.MarshalJSON(&authState)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal updated auth state: %s", err)
+		}
+
+		// and fund the account
+		// unmarshal bank module state
+		bankState := bankTypes.GenesisState{}
+		if err := codec.UnmarshalJSON(appState[bankTypes.ModuleName], &bankState); err != nil {
+			return nil, fmt.Errorf("failed to unmarshal app_state.bank: %s", err)
+		}
+
+		balance := bankTypes.Balance{
+			Address: memberAddress,
+			Coins: []sdk.Coin{
+				sdk.Coin{
+					Denom:  "ukava",
+					Amount: math.NewInt(100000000000000),
+				},
+			},
+		}
+
+		bankState.Balances = append(bankState.Balances, balance)
```

Comment:
also need to update total supply here? though i think you mentioned that irl already 😄